### DISCUSSION TITLE
[Entry]: Unofficial patches for Ripcord

### DISCUSSION THREAD

**Discussion by Jevil7452** [Discord-Client-Encyclopedia-Management/Discord3rdparties#205] [Ideas]
### Name

patcher/ripcord-audio-hook

### Link

https://github.com/geniiii/patcher and https://github.com/geniiii/ripcord-audio-hook

### Features

Patches or hooks Ripcord's binary for some extra features (See further comments)

### Language(s)

C

### Development Status

On hiatus

### Further comments

Last commit was 6 months ago to the audio hook repo, so I am marking it as "On hiatus"
Features of the binary patcher:
-See deleted messages
-Send messages in channels with a cooldown enabled
Features of the hook:
-Fix voice channel stuff ("reading RTP header extensions larger than 8 bytes as voice data, causing garbled audio in voice chats", "sending and receiving an old (?) version of the IP discovery packet, causing it to get stuck on "Routing..." when attempting to join voice channels" and "not showing "Stage" voice channels")
-Fix image previews

**Comment by KrstlSkll69** [comment]
Hello, I'd like to put in my opinion here. As these two patches are talked about that in the Ripcord Server. I think that they should be put on the list _(but as a foot note)_ and as such I have opened a Pr to add them
https://github.com/Discord-Client-Encyclopedia-Management/Discord3rdparties/pull/218

**Reply by lexyclear** [reply]
i agree

**Comment by lexyclear** [comment]
this has already been done so closing this one.
